redis/src/commands/json.rs

```diff
@@ -0,0 +1,135 @@
+// can't use rustfmt here because it screws up the file.
+#![cfg_attr(rustfmt, rustfmt_skip)]
+use crate::cmd::{cmd, Cmd, Iter};
+use crate::connection::ConnectionLike;
+use crate::pipeline::Pipeline;
+use crate::types::{FromRedisValue, RedisResult, ToRedisArgs};
+
+#[cfg(feature = "cluster")]
+use crate::commands::ClusterPipeline;
+
+use serde::ser::Serialize;
+
+implement_commands! {
+    'a
+	
+	/// Append the JSON `value` to the array at `path` after the last element in it.
+	#[cfg_attr(docsrs, doc(cfg(feature = "json")))]
```

Comment:
Is there any reason for these attributes, given that they're all within a module that's conditional on the feature?

Reply:
You make a good point, will fix